modules/minimp3/doc_classes/ResourceImporterMP3.xml

```diff
@@ -13,15 +13,15 @@
 	</tutorials>
 	<members>
 		<member name="bar_beats" type="int" setter="" getter="" default="4">
-			The number of bars within a single beat in the audio track. This is only relevant for music that wishes to make use of interactive music functionality (not implemented yet), not sound effects.
+			The number of bars within a single beat in the audio track. This is only relevant for music that wishes to make use of interactive music functionality, not sound effects.
 			A more convenient editor for [member bar_beats] is provided in the [b]Advanced Import Settings[/b] dialog, as it lets you preview your changes without having to reimport the audio.
 		</member>
 		<member name="beat_count" type="int" setter="" getter="" default="0">
-			The beat count of the audio track. This is only relevant for music that wishes to make use of interactive music functionality (not implemented yet), not sound effects.
+			The beat count of the audio track. This is only relevant for music that wishes to make use of interactive music functionality, not sound effects.
 			A more convenient editor for [member beat_count] is provided in the [b]Advanced Import Settings[/b] dialog, as it lets you preview your changes without having to reimport the audio.
 		</member>
 		<member name="bpm" type="float" setter="" getter="" default="0">
-			The Beats Per Minute of the audio track. This should match the BPM measure that was used to compose the track. This is only relevant for music that wishes to make use of interactive music functionality (not implemented yet), not sound effects.
+			The Beats Per Minute of the audio track. This should match the BPM measure that was used to compose the track. This is only relevant for music that wishes to make use of interactive music functionality, not sound effects.
```

Comment:
```suggestion
			The beats per minute of the audio track. This should match the BPM measure that was used to compose the track. This is only relevant for music that wishes to make use of interactive music functionality, not sound effects.
```
Should "Beats Per Minute" be capitalized here? (Same thing in ResourceImporterOggVorbis)

Reply:
It really shouldn't. All descriptions about interactive music are somewhat shoddy, but this can be fixed too for now.